service/authorization/authorization_test_structures.go

```diff
@@ -0,0 +1,334 @@
+package authorization
```

Comment:
Maybe we can move this to authorization_test? I guess we'd have to move authorization_test too.